js/main.js

```diff
@@ -123,8 +120,8 @@ function setupThenOpenMainWindow() {
         // as the app is launched as a root user we don't get
         // access to the config file
         let launchOnStartup = process.argv[3];
-        updateUserConfigMac()
-            .then(setStartup(launchOnStartup))
+        setStartup(launchOnStartup)
+            .then(() => updateUserConfigMac(process.argv[2]))
```

Comment:
With the post install script, we pass in an argument to get the global Symphony.config path file (i.e. /Applications/Symphony.app/Contents/config/Symphony.config). We unfortunately don't get this directly from the app as it'll be run from a temporary path and any relative path for the config file doesn't work and we end up manipulating the incorrect config file.

Regardless, I've abstracted this to a variable for better clarity.